app/lib/frontend/static_files.dart

```diff
@@ -172,30 +158,32 @@ class StaticUrls {
     };
   }
 
-  /// A hashed version of the static assets referenced in [hashedFiles].
+  /// A hashed version of the static assets.
   Map<String, String> get assets {
     if (_assets == null) {
       _assets = <String, String>{};
-      for (String hashedFile in hashedFiles) {
+      for (String requestPath in staticFileCache.keys) {
+        final inStatic = requestPath.startsWith('$_defaultStaticPath/');
+        // Removing the /static/ prefix from the keys in order to make them
+        // shorter in the templates.
+        final hashedFile = inStatic
+            ? requestPath.substring(_defaultStaticPath.length + 1)
```

Comment:
Why can you remove `/static/`? are these always being used as relative?

Reply:
https://github.com/dart-lang/pub-dev/blob/master/app/lib/frontend/templates/views/layout.mustache#L34-L36

```{{{static_assets.js__script_dart_js}}}``
vs
```{{{static_assets.static__js__script_dart_js}}}```

Reply:
Please illustrate that in the doc comment... :)

it makes a lot of sense though :)